client/src/components/Reviews/Review.jsx

```diff
@@ -1,26 +1,71 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { SingleReviewStars } from '../styles/reviewstyles/SingleReviewStars.styled.js';
+import { ThumbNail } from '../styles/reviewstyles/ThumbNail.styled.js';
+import { Background } from '../styles/reviewstyles/FormBackground.styled.js';
+import { LargePhoto } from '../styles/reviewstyles/LargePhoto.styled.js';
 
-const Review = ({review}) => {
+const Review = ({ review, StarReview }) => {
 
-  // "review_id": 1274559,
-  // "rating": 3,
-  // "summary": "this is kevin's summary",
-  // "recommend": true,
-  // "response": null,
-  // "body": "let''s sogooooo get it together",
-  // "date": "2022-05-26T00:00:00.000Z",
-  // "reviewer_name": "mynickname",
-  // "helpfulness": 3,
-  // "photos": []
+  const [restriction, toggle] = useState(false);
```

Comment:
naming convention is usually stateName and then setStateName to make it clear which state variable you are changing